### ISSUE TITLE
Expand the option for vertical video crop also to pagetype "video player"

### ISSUE THREAD

**Issue by rufposten** [codevise/pageflow#1203]
Currently, the field of view (viewport) settings and hence the ability to view 16:9 media on a smartphone trimmed in 9:16 is currently only available for the background pagetype. Here even the background videos get trimmed so they fill the whole screen. 

Would be great if also the pagetype "video player" would have an option to change the aspect ratio of the videos on a smartphone accordingly. Should be optional, as not all videos are suitable for trimming. 

![theres](https://user-images.githubusercontent.com/49686764/63263981-462adb80-c28a-11e9-98f9-bcd229e74605.png)

**Comment by tf** [comment]
I agree this would be nice to have. As a prerequisite one would need to change the way the video page works on iOS. There we still rely on Safari's built in video player which opens in a modal instead of using our own player controls like on Android (see screenshot above). Once that change is done, adding the option should not be to difficult since the video page uses the same video component that is also used for background videos.

We currently have no plans to work on this change. If you should be interested to dive into the technical details, I'd be happy to give some pointers.

**Comment by rufposten** [comment]
Nice to hear that it is easy and appreciated! I already looked into the code and discovered that it's impossible to do in script-kiddie-mode without understanding ruby. But probably someone else?

**Comment by rufposten** [comment]
Now after introducing vertical background video as a second upload option and portrait inline video in Next I wonder if this issue regarding viewport and aspect for Pageflow Classic would be easier to solve?
